shared/fetcher.js

```diff
@@ -187,6 +187,7 @@ Fetcher.prototype.fetchFromApi = function(spec, options, callback) {
         respOutput = JSON.stringify(resp);
         err = new Error("ERROR fetching model '" + fetcher.modelUtils.modelName(model.constructor) + "' with options '" + JSON.stringify(options) + "'. Response: " + respOutput);
         err.status = resp.status;
```

Comment:
I'd say to use underscore and a whitelist instead, this way we can easily add or remove things from the whitelist.

```js
_.extend(err, _.pick(resp, [
  'status', 'statusText'
]));
```